### PR TITLE
DMU: 3 kicker cards

### PR DESCRIPTION
**Pull request by Suthro**
DMU: Added card scripts for 3 DMU cards

- Elvish Hydromancer
- Joint Exploration
- Vineshaper Prodigy

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/vineshaper_prodigy.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Vineshaper Prodigy
+ManaCost:1 G
+Types:Creature Elf Druid
+PT:2/2
+K:Kicker:1 U
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self+kicked | Execute$ TrigDig | TriggerDescription$ When CARDNAME enters the battlefield, if it was kicked, look at the top three cards of your library. Put one of them into your hand and the rest on the bottom of your library in any order.
+SVar:TrigDig:DB$ Dig | DigNum$ 3 | ChangeNum$ 1
+Oracle:\nKicker {1}{U} (You may pay an additional {1}{U} as you cast this spell.)\nWhen Vineshaper Prodigy enters the battlefield, if it was kicked, look at the top three cards of your library. Put one of them into your hand and the rest on the bottom of your library in any order.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/vineshaper_prodigy.txt:8]: Oracle:\n

\n isn't needed here

File: forge-gui/res/cardsfolder/upcoming/elvish_hydromancer.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Elvish Hydromancer
+ManaCost:2 G
+Types:Creature Elf Wizard
+PT:3/2
+K:Kicker:3 U
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self+kicked | Execute$ TrigCopy | TriggerDescription$ When CARDNAME enters the battlefield, if it was kicked, create a token that's a copy of target creature you control.
+SVar:TrigCopy:DB$ CopyPermanent | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control
+Oracle:Kicker {3}{U} (You may pay an additional {3}{U} as you cast this spell.)\nWhen Elvish Hydromancer enters the battlefield, if it was kicked, create a token that's a copy of target creature you control.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/elvish_hydromancer.txt:8]: Maybe DeckHints:Ability$Token ?

File: forge-gui/res/cardsfolder/upcoming/joint_exploration.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Joint Exploration
+ManaCost:1 U
+Types:Instant
+K:Kicker:G
+A:SP$ Scry | ScryNum$ 2 | SubAbility$ DBDraw | SpellDescription$ Scry 2, then draw a card.
+SVar:DBDraw:DB$ Draw | NumCards$ 1 | SubAbility$ DBChangeZone
+SVar:DBChangeZone:DB$ ChangeZone | Condition$ Kicked | Origin$ Hand | Destination$ Battlefield | Optional$ You | ChangeType$ Land | ChangeNum$ 1 | SpellDescription$ If this spell was kicked, you may put a land card from your hand onto the battlefield.
+Oracle:Kicker {G} (You may pay an additional {G} as you cast this spell.)\nScry 2, then draw a card. If this spell was kicked, you may put a land card from your hand onto the battlefield.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/joint_exploration.txt:8]: DeckHints:Color$Green

File: forge-gui/res/cardsfolder/upcoming/vineshaper_prodigy.txt (hunk 1 above)
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/vineshaper_prodigy.txt:8]: DeckHints:Color$Blue

**Comment by Suthro** [comment]
@paulsnoops Made the requested changes, thank you!

**Review by paulsnoops — APPROVED** [review]

**Comment by paulsnoops** [comment]
> @paulsnoops Made the requested changes, thank you!

Nah, thank you for scripting them! :)

**Pull request merged**